using-mobile-extensions/adobe-mobileservices/mobileservices-api-reference.md

```diff
@@ -0,0 +1,78 @@
+# Mobile Services Event Reference
+
+## Track Deep link  <a id="trackdeeplink"></a>
+
+To track a Deep Link or Universal Link which confirms to Adobe's required format, which can either be a link created on Adobe Mobile Services UI, or generated by another vendor.
```

Comment:
- Please lower case "Deep Link" --> "deep link"

- I'm not quite sure what the sentence is saying, but here's a rewrite suggestion:
"To track a deep link or Universal Link, Adobe's format requires that the link be created in the Adobe Mobile Services UI or by another vendor."

Does this work?